spec/models/order_spec.rb

```diff
@@ -2,7 +2,16 @@
 
 RSpec.describe Order, type: :model do
   let(:user) { Fabricate(:user) }
-  let(:order) { Order.create(user_id: user.id, status: 0) }
+  let(:item) { item = Fabricate.build(:item)
+               item.categories << category
+               item.save
+               item}
+  let(:category) { Fabricate(:category) }
+  let(:cart) { Cart.new("#{item.id}" => 1)}
+  let(:order) { order = Order.new(user_id: user.id, status: 0)
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.